Rubberduck.Main/ComClientLibrary/UnitTesting/FakesProvider.cs

```diff
@@ -37,6 +37,30 @@ public void StartTest()
             CodeIsUnderTest = true;
         }
 
+        public static void SuspendFake(String typename)
+        {
+            foreach (var fake in ActiveFakes.Values)
+            {
+                if (fake.GetType().Name == typename)
+                {
+                    fake.DisableHook();
+                    return;
+                }
+            }
+        }
+
+        public static void ResumeFake(String typename)
+        {
+            foreach (var fake in ActiveFakes.Values)
+            {
+                if (fake.GetType().Name == typename)
+                {
+                    fake.DisableHook();
```

Comment:
The implementation is identical to that of `SuspendFake`, is that intended?

Reply:
Definitely wasn't intended! I'll fix and also enhance the test case to cover the reenabling of the hook.